{
  throw new RuntimeException("Test exception");
}
